src/render.js

```diff
@@ -0,0 +1,11 @@
+const render = (arr) => {
+  const container = document.querySelector('ul');
+  arr.forEach((game, index) => {
+    const li = document.createElement('li');
+    li.textContent = `${game.user}: ${game.score}`;
+    li.id = index;
+    container.appendChild(li);
+  });
+};
+
+export default render;
```

Comment:
- [ ] There is an issue with your rendering. When I click the `Refresh` button, it appends the fetched scores to the existing ones, resulting in multiple scores. I would recommend you fix this.

  **SUGGESTION:** You can empty the container element by setting its innerHTML to an empty string before rendering the fetched scores, like so:
  ```
  const render = (arr) => {
    const container = document.querySelector('ul');
    container.innerHTML = ''
    ...
  ```

Reply:
Thank you for the feedback. 
Will implement the required changes.
Thank you.